crates/astria-conductor/src/executor/tests.rs

```diff
@@ -140,7 +153,10 @@ struct MockEnvironment {
     executor: Executor,
 }
 
-async fn start_mock(disable_empty_block_execution: bool) -> MockEnvironment {
+async fn start_mock(
+    disable_empty_block_execution: bool,
```

Comment:
Just noting here that this pr to remove the disable empty block execution is in flight:
https://github.com/astriaorg/astria/pull/556

Reply:
yeah i can fix conflicts if it gets merged first